src/lib/mdns/minimal/Server.h

```diff
@@ -28,6 +28,20 @@
 namespace mdns {
 namespace Minimal {
 
+/// These are addresses that are used for MDNS broadcasts
+struct BroadcastIpAddresses
+{
+    chip::Inet::IPAddress ipv6;
+    chip::Inet::IPAddress ipv4;
+
+    BroadcastIpAddresses()
+    {
+        chip::Inet::IPAddress::FromString("FF02::FB", ipv6);
+        chip::Inet::IPAddress::FromString("224.0.0.251", ipv4);
+    }
+};
+extern const BroadcastIpAddresses kBroadcastIpAddresses;
```

Comment:
Please don't do this. BroadcastIpAddresses() isn't constexpr, this is going to generate a static initializer that parses a string which will run in the initial thread before anything else happens in the program. It's a source of performance problems, and also defeats feature toggles in the sense that we'll be forced to spend runtime initializing parts of CHIP at runtime by merely linking the code into the program.